### DISCUSSION TITLE
Comments not enabled

### DISCUSSION THREAD

**Discussion by lauravignoli** [requarks/wiki#3105] [Help / Questions] [answered]
Hi, 

I'm trying to enable comments for my site but they are not showing.

I added read and write comments permissions for groups and page rules and the default comments is clicked in the Comments section:

![enabled](https://user-images.githubusercontent.com/1198827/108391954-e0bb0080-721a-11eb-813c-b1abca2dbee4.png)

Does anyone have any advice?

Thanks,
Laura

**Comment by gmasil** [comment, marked as answer]
Do you have comments enabled in admin area -> general?
![image](https://user-images.githubusercontent.com/54176035/108545140-b125fa00-72e7-11eb-8e46-3fb43058db93.png)

**Reply by lauravignoli** [reply]
Thanks so much!! That did the trick ;)
